### PR TITLE
Issue 304/remove unused code for mvp

### LINKED ISSUE #304: Remove all pages out of scope of MVP
Remove:
* Pages
* Components
* Locale strings
* Layouts
* Hooks
* Integration tests
* Cypress Fixtures

Also remove:
* Adope Spectrum and Provider

Pages to remove:
* /o/*
* /my/*
* /organize/
  * people
  * areas
  * campaigns
    * */insights

### PR DESCRIPTION
**Pull request by djbusstop**
Resolves #304

### CODE REVIEW

File: src/pages/404.tsx [hunk 1]
```diff
@@ -22,22 +22,16 @@ export default function Custom404() : JSX.Element {
...
-                    <Heading level={ 1 }>
-                        <Msg id="pages.404.pageNotFound"/>
-                    </Heading>
-                    <Link data-testid="back-home-link">
-                        <a href="/">
-                            <Msg id="pages.404.backToHomePage"/>
-                        </a>
-                    </Link>
-                </Flex>
-            </Content>
```
Comment (djbusstop) [inline src/pages/404.tsx:40]: Need to make issue to make nice style for this page.

File: next.config.js [hunk 2]
```diff
@@ -1,12 +1,7 @@
 const withPlugins = require('next-compose-plugins')
 const withCSS = require('@zeit/next-css')
-const withTM = require('next-transpile-modules')([
-    '@adobe/react-spectrum',
-    '@spectrum-icons/.*',
-    '@react-spectrum/.*'
-])
 
-module.exports = withPlugins([withCSS, withTM], {
+module.exports = withPlugins([withCSS], {
```
Comment (richardolsson) [inline next.config.js:4]: Is `withCSS` still required with the way that CSS is working in MUI? I believe we only added it for the sake of Spectrum.

File: src/pages/404.tsx (hunk 1 above)
Comment (djbusstop) [reply src/pages/404.tsx:40]: Issue made: #306

**Review by richardolsson — APPROVED** [review]
It all seems to be working and changes look good to me. I found one candidate for deletion below. If I'm wrong about it just go ahead and merge, otherwise maybe there's more to be deleted. 😊 

I've also created a branch called `before-304-cleanup` which we can keep around for a while in case we ever want to easily find a revision where the code still exists.

File: next.config.js (hunk 2 above)
Comment (djbusstop) [reply next.config.js:4]: oh interesting! I'll check that out. Was wondering what "zeit" was other than a pretty decent centrist newspaper in germany

Comment (djbusstop) [reply next.config.js:4]: Nice you're right, next supports css out of the box and this was unneeded, and giving warnings in the console. 2 birds, 1 stone.

Comment (djbusstop) [reply next.config.js:4]: Great idea for the branch, thank you!

File: next.config.js [hunk 3]
```diff
@@ -1,9 +1,4 @@
-const withPlugins = require('next-compose-plugins')
-
-module.exports = withPlugins([], {
-  webpack: (config) => {
```
Comment (richardolsson) [inline next.config.js]: Great! I think this was only there because of linting that used to run in the Webpack pipeline but was then moved outside of the build step (58a73d4ab0e655d6cc019aecf052b2b1ca227132).

**Review by richardolsson — APPROVED** [review]
Great! 🟢 Green light for merging (or "purging", amirite).

File: next.config.js (hunk 3 above)
Comment (djbusstop) [reply next.config.js]: makes sense! hell yea. Let's go!

File: src/theme.ts [hunk 4]
```diff
@@ -1,8 +1,8 @@
 import { createElement } from 'react';
-import { createMuiTheme } from '@material-ui/core/styles';
+import { createTheme } from '@material-ui/core/styles';
 
 // Create a theme instance.
-const theme = createMuiTheme({
+const theme = createTheme({
     overrides: {
```
Comment (nehahirve) [inline src/theme.ts:6]: Hey sorry that I didn't get a chance to look at this before! Why did you change this? I have to change it back in my current branch in order for it to compile
